circle.yml

```diff
@@ -22,24 +21,24 @@ machine:
     - rabbitmq-server
 
 dependencies:
-  override:  # Let's install Mongo 3 manually because circle doesn't support mongo3 yet
-    - sudo apt-get purge mongodb-org*
-    - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
-    - echo "deb http://repo.mongodb.org/apt/ubuntu trusty/mongodb-org/3.2 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
-    - sudo apt-get update
-    - sudo apt-get install -y mongodb-org
-    - sudo service mongod restart
   cache_directories:
     - ~/.cache/pip
   pre:
     - sudo .circle/configure-services.sh
     - sudo .circle/add-itest-user.sh
     - sudo apt-get -y install parallel jq
-  post:
-    - cd ${ST2_REPO_ROOT_DIR} && hostname && make compile requirements
+  # Let's install Mongo 3 manually because circle doesn't support mongo3 yet
+    - sudo .circle/install-mongo.sh
+    - sudo pip install codecov
+  override:
+    - make compile requirements
```

Comment:
That's how it was done previously and I don't really understand it deep enough to mess with it. Can you elaborate?